### PR TITLE
convert image registry to k8s.gcr.io

### PR DESCRIPTION
**Pull request by zouyee**
convert image registry to k8s.gcr.io

/assign @mikesplain

### CODE REVIEW

File: addons/metrics-server/v1.8.x.yaml [hunk 1]
```diff
@@ -136,7 +136,7 @@ spec:
         emptyDir: {}
       containers:
       - name: metrics-server
-        image: gcr.io/google_containers/metrics-server-amd64:v0.3.1
+        image: k8s.gcr.io/metrics-server-amd64:v0.3.0
```
Comment (rdrgmnzs) [inline addons/metrics-server/v1.8.x.yaml]: @zouyee any reason we're downgrading from v0.3.1 to v0.3.0 here?

Comment (zouyee) [reply addons/metrics-server/v1.8.x.yaml]: https://github.com/kubernetes-incubator/metrics-server/pull/148
https://github.com/kubernetes-incubator/metrics-server/pull/138

Comment (mikesplain) [reply addons/metrics-server/v1.8.x.yaml]: From what I can tell, that bug was fixed in 0.3.1 and the image manifest simply wasn't updated in the release, but is still the latest and suggested version. I'm not clear on why we want to roll back? What am I missing 😄

Comment (zouyee) [reply addons/metrics-server/v1.8.x.yaml]: oh, my mistake

**Comment by justinsb** [comment]
So this looks good, but could we fix up the version number?

**Comment by justinsb** [comment]
(PS thanks @zouyee !)

**Comment by justinsb** [comment]
Ah - looks like I did this as part of a sweep in #6623 

Thanks for the PR though, and sorry we didn't get this one in.

**Pull request closed without merging**